src-docs/src/views/page/page_example.js

```diff
@@ -229,25 +226,24 @@ export const PageExample = {
           />
         </>
       ),
-      demo: (
-        <PageDemo>
-          {(Button, Content, SideNav, showTemplate) =>
-            showTemplate ? (
-              <PageRestricingWidthTemplate
-                button={<Button />}
-                content={<Content />}
-                sideNav={<SideNav />}
-              />
-            ) : (
-              <PageRestricingWidth
-                button={<Button />}
-                content={<Content />}
-                sideNav={<SideNav />}
-              />
-            )
-          }
-        </PageDemo>
-      ),
+      demoRoute: {
+        slug: 'restricting-page-width',
+        demo: (
+          <PageDemo
+            slug="restricting-page-width"
+            pattern={PageRestrictingWidth}
+            template={PageRestrictingWidthTemplate}
+            fullscreen
+          />
+        ),
+        placeholder: (
+          <PageDemo
+            slug="restricting-page-width"
+            pattern={PageRestrictingWidth}
+            template={PageRestrictingWidthTemplate}
+          />
+        ),
+      },
```

Comment:
This syntax is clear to me 🎉 

But what is `placeholder`?

Reply:
`placeholder` is an optional field, easier to define what it is by demonstrating what happens when it is left out, as is the case with the full collapsible nav pattern which gets the _Goto Fullscreen Example_ button:

<img width="252" alt="Screen Shot 2021-04-07 at 12 17 54 PM" src="https://user-images.githubusercontent.com/313125/113914892-4e0afc80-979b-11eb-9fd5-dd51e89cbdfe.png">

`placeholder` defines what content to display in that example viewport instead of that button. Very open to naming suggestions.

Reply:
Hmm ok, then maybe I'd actually rename both to be:
```suggestion
      demoRoute: {
        slug: 'restricting-page-width',
        fullscreen: (
          <PageDemo
            slug="restricting-page-width"
            pattern={PageRestrictingWidth}
            template={PageRestrictingWidthTemplate}
            fullscreen
          />
        ),
        demo: (
          <PageDemo
            slug="restricting-page-width"
            pattern={PageRestrictingWidth}
            template={PageRestrictingWidthTemplate}
          />
        ),
      },
```

Reply:
Or... If `demoRoute` can still be used in conjunction with `demo` at the top level. You could simply do:

```tsx
      demo: (
        <PageDemo />
      ),
      fullScreen: {
        slug: 'restricting-page-width',
        demo: (
          <PageDemo  />
        ),
      },
```

Reply:
Oh I like that!

Reply:
Pushed this change up

Reply:
Pushed the change to use `demo` instead of `demoRoute.placeholder`